src/query/storages/table-meta/src/meta/versions.rs

```diff
@@ -163,9 +163,13 @@ mod converters {
         type Error = ErrorCode;
         fn try_from(value: u64) -> Result<Self, Self::Error> {
             match value {
-                1 | 2 => Err(ErrorCode::DeprecatedIndexFormat(
+                1 => Err(ErrorCode::DeprecatedIndexFormat(
```

Comment:
Looks better to define constant instead the int `1` `2` `3`.

Reply:
this is already the lowest level now and it's the same logic as before.

We can define constant field name  `const VersionV1 = 1`, but it's too redundant